src/main/request.ts

```diff
@@ -135,6 +135,25 @@ export class Request {
         });
     }
 
+    async getFile(url: string, type: string, options: RequestOptions = {}) {
+        const { body, query, headers } = options;
+        const res = await this.send('get', url, {
+            headers: {
+                'content-type': type,
+                ...headers,
+            },
+            query,
+            body: body ? JSON.stringify(body) : null,
+        });
+        const { status } = res;
+        if (status === 204 || res.headers.get('content-length') === '0') {
+            return null;
+        }
+        if (res.ok) {
+            return res.blob();
+        }
```

Comment:
We don't really need to check for `res.ok`, since the `send` method already does this, and if it's not ok it throws an error